client/src/views/pages/search.js

```diff
@@ -132,12 +132,21 @@ export default function (props) {
               <>
                 <CardBody className="text-center border-bottom">
                   <h4>Steam Profile</h4>
-                  {data.steamUser.name && data.steamUser.avatarFull && (
+                  {data.steamUser.name && data.steamUser.avatarFull ? (
```

Comment:
`.name` shouldn't be considered here as this if statement should only be referring to whether they have an avatar or not?